execution_engine/src/core/runtime/auction_internal.rs

```diff
@@ -136,6 +136,10 @@ where
     fn blake2b<T: AsRef<[u8]>>(&self, data: T) -> [u8; BLAKE2B_DIGEST_LENGTH] {
         crypto::blake2b(data)
     }
+
+    fn auction_bids_allowed(&self) -> bool {
```

Comment:
Maybe for consistency?

```suggestion
    fn allow_auction_bids(&self) -> bool {
```